scripts/docker_compose_config.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
Is there documentation for the .json file format?

Reply:
Added better documentation into scripts readme